.github/dependabot.yaml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: "nuget"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 5
```

Comment:
is there a reason we are limiting this to 5?

Reply:
No we don't need this anymore. Removing it.